SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a security issue, please use the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/nt4f04uNd/sweyer/security/advisories/new) tab.
```

Comment:
I think this is only intended to be used by maintainers and other people can't create anything there. The link is definitely not accessible without logging in, so people without a GitHub account wouldn't be able to report anything.

Reply:
Maybe just put an email address there instead?

Reply:
Thanks for adding the email. I checked and according to [the docs](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability) it is possible for non-admins to report vulnerabilities, so this is perfect.